collector.spec

```diff
@@ -0,0 +1,54 @@
+%global goipath         github.com/os-observability/redhat-opentelemetry-collector
+
+Version:                0.92.0
```

Comment:
Can we use the verison defined in the `Makefile` here?

Reply:
Yeah, you can use `sed`. There is usually also some automation for bumping the version. For example:

- https://github.com/pykickstart/pykickstart/blob/master/pykickstart.spec.in#L7
- https://github.com/pykickstart/pykickstart/blob/master/Makefile#L1
- https://github.com/pykickstart/pykickstart/blob/master/Makefile#L117-L128

However, I would explore what packit can do first (https://packit.dev/). I am not familiar with latests trends and I remember some discussion about managing the spec file. However, that might have beeen the downstream spec file.